packages/core/apps/core-apps-server-internal/BUILD.bazel

```diff
@@ -0,0 +1,142 @@
+load("@npm//@bazel/typescript:index.bzl", "ts_config")
+load("@build_bazel_rules_nodejs//:index.bzl", "js_library")
+load("//src/dev/bazel:index.bzl", "jsts_transpiler", "pkg_npm", "pkg_npm_types", "ts_project")
+
+PKG_DIRNAME = "core-apps-server-internal"
+PKG_REQUIRE_NAME = "@kbn/core-apps-server-internal"
+
+SOURCE_FILES = glob(
+  [
+    "**/*.ts",
+    "src/**/*.css",
```

Comment:
@elastic/kibana-operations is there a trick to serving the assets directly from the package so that we don't need to keep them in `src/core`?

Reply:
WDYM by 'serving' here exactly?

To my knowledge, there are two usages of these files:

The correct one:

https://github.com/elastic/kibana/blob/8176ac74f08c51838a620116ae33d0d433bf05de/src/core/server/core_app/core_app.ts#L206

which should work out of the box when the assets are moved in the package (right?)

And that scss import, that I didn't even knew about:

https://github.com/elastic/kibana/blob/bebec37f0435f292f19834d49c8db0006f85fcbb/x-pack/plugins/canvas/shareable_runtime/index.ts#L9

You're referring to the second one here, right?

Reply:
yes, I'm referring to the second one.